### PR TITLE
fix: add support for new private repo in org

### PR DESCRIPTION
**Pull request by jainpawan21**
Issue Number: #2490 

## PR Details

Add support for create new private repo in org.

## PR Checklist
- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by yuval-hazaz** [comment]
@tupe12334 @alexbass86 please review

File: packages/amplication-server/.gitignore [hunk 1]
```diff
@@ -9,5 +9,6 @@ coverage/
 prisma/migrations/dev/
 artifacts/
 .env.local
+.env
```
Comment (tupe12334) [inline packages/amplication-server/.gitignore:12]: @jainpawan21 Why did you add the .env file here?

**Comment by jainpawan21** [comment]
@tupe12334, as mentioned in docs, I created a new file named .env.local and copied content of .env in it, then I tried to use GitHub services, but those were not working, 
Github services were only working with .env variables. 
That's why I thought of adding it in gitignore and created a new file `.env.example`

**Comment by yuval-hazaz** [comment]
@tupe12334 please instruct @jainpawan21 how to proceed from here

**Comment by tupe12334** [comment]
@jainpawan21  Please remove from this pr all the stuff that is irrelevant to the issue like the .env stuff and the ci test for the git-service isn't passing

**Review by tupe12334 — APPROVED** [review]

**Comment by tupe12334** [comment]
@jainpawan21 I saw that you update the pr but you didn't update the git-service yet

**Comment by tupe12334** [comment]
> @yuval-hazaz What should we do with this one?

We need to fix the location of the button, I'm sure that @alexbass86 will not approve that

**Comment by yuval-hazaz** [comment]
@tupe12334 can you please send a link or a screenshot?

**Comment by tupe12334** [comment]
This is in private mode
Screen Shot 2022-07-25 at 15 40 49

And this in public

Screen Shot 2022-07-25 at 15 40 43

@yuval-hazaz  and @alexbass86

**Comment by alexbass86** [comment]
> 
Hi, I designed a new popup for this feature. Please make sure it's aligned.

a4

Figma:
https://www.figma.com/file/8Q9kB3b29FCh9ttVmOHm9u/Concept---Private-repo?node-id=1601%3A5670

**Review by alexbass86 — CHANGES_REQUESTED** [review]
I posted the design for this popup, and please make sure it's aligned

**Comment by yuval-hazaz** [comment]
@alexbass86 thanks for the new design. 
I will approve this PR as it is and move forward
please open a new issue to use the new design.

@tupe12334 please merge this PR

**Review by yuval-hazaz — APPROVED** [review]

**Comment by alexbass86** [comment]
> @alexbass86 thanks for the new design. I will approve this PR as it is and move forward please open a new issue to use the new design.
> 
> @tupe12334 please merge this PR

done [new issue](https://github.com/amplication/amplication/issues/3250)

**Comment by tupe12334** [comment]
@jainpawan21 Do you want to work on the update or I will take it from here?

**Comment by jainpawan21** [comment]
> @jainpawan21 Do you want to work on the update or I will take it from here?

Hi @tupe12334, yes you can take it from there as there is new issue opened

**Pull request merged**